pyVor/tests/testPredicates.py

```diff
@@ -122,6 +122,26 @@ def test_incircle_homogeneous(self):
         self.assertEqual(incircle(*inf_triangle[::-1], Point(-1, 1, 1)),
                          -1)
 
+        # Tricky cases:
+        self.assertEqual(incircle(
+            Point(-1, -1, 0), Point(-3, 2, 1), Point(1, 0, 0), Point(0, 1, 0)),
+                         1)
+        self.assertEqual(incircle(
+            Point(-3, 2, 1), Point(-1, -1, 0), Point(1, 0, 0), Point(0, 1, 0)),
+                         -1)
+
+        # The following tests are based on pencil-and-paper drawings
+        # as well as hours spent debugging the DelaunayTriangulation.
+
+        self.assertEqual(incircle(
+            Point(1, 0, 0), Point(-0.6, 3.2, 1), Point(3.2, 2.1, 1),
+            Point(-2, 0, 1)),
+                         -1)
+
+        self.assertEqual(incircle(
```

Comment:
This test will also fail, due to the fact that the Points are of the same dimension. What was the test that you were going for here?